lib/bpl/transformation/shadowing.rb

```diff
@@ -26,6 +26,8 @@ def shadow_copy(node)
         shadow
       end
 
+      def shadow_expr_eq(expr) "#{expr} == #{shadow_copy(expr)}" end
```

Comment:
Any good reason not to just define `shadow_eq` this way, rather than having both `shadow_eq` and `shadow_expr_eq`?